File: src/main/java/no/nav/fo/veilarbregistrering/oppgave/resources/OppgaveMapper.java
package no.nav.fo.veilarbregistrering.oppgave.resources;

import no.nav.fo.veilarbregistrering.oppgave.OppgaveResponse;
import no.nav.fo.veilarbregistrering.oppgave.OppgaveType;

class OppgaveMapper {

    private OppgaveMapper() {
    }

    static OppgaveDto map(OppgaveResponse oppgaveResponse, OppgaveType oppgaveType) {
        return new OppgaveDto(oppgaveResponse.getId(), oppgaveResponse.getTildeltEnhetsnr(), oppgaveType);
    }
}

File: src/main/java/no/nav/fo/veilarbregistrering/oppgave/resources/OppgaveMapper.kt
package no.nav.fo.veilarbregistrering.oppgave.resources

import no.nav.fo.veilarbregistrering.oppgave.OppgaveResponse
import no.nav.fo.veilarbregistrering.oppgave.OppgaveType

internal object OppgaveMapper {
    @JvmStatic
    fun map(oppgaveResponse: OppgaveResponse, oppgaveType: OppgaveType): OppgaveDto =
            OppgaveDto(oppgaveResponse.id, oppgaveResponse.tildeltEnhetsnr, oppgaveType)
}